### ISSUE TITLE
Add Webhook Subscriber

### ISSUE THREAD

**Issue by martinjt** [ks888/LambStatus#111]
Specifically, I'm thinking that posting into slack would be a valueable addon to this.

**Comment by ks888** [comment]
Yes, the webhook subscribing option will be a great addition. Until the feature is supported, the workaround using [ifttt](https://ifttt.com/connect/email/slack) may work well.
